recipes

```diff
@@ -1 +1 @@
-Subproject commit cdafac130fd454610d1fa2cb53b061b17b64e2ce
+Subproject commit 80f8496bf811a8fb93e9a961141c4868f66608fe
```

Comment:
Hi @mcmxcdev ! Thank you for this PR. I think you staged and committed a change in the recipes submodule (that is probably why the builds are failing)

Reply:
Ugh, had to rush, always happens to me unintentionally when I 'git add -A'.

I will be afk for a while, so could you remove it from the PR and get it merged?

Reply:
Solved it by undoing the changes and force-pushing again in https://github.com/ferdium/ferdium-app/pull/1385/commits